checkmanga.py

```diff
@@ -2,78 +2,128 @@
 import requests
 import sqlite3
 import sys
-
+import datetime
 #'mycheckmanga.db' is the name of the exmaple database included with this code.
 
-
-def db_get_entry_info(title, cur):
-    """Input: Manga title, and cursor object 
-    Output: Returns a dictionary of the manga's info from the db, keys to the dict are columns of the db."""
-    cur.execute("SELECT * FROM manga WHERE Title = (?)", [title])
-    entry = cur.fetchone()
-    return dict(entry) 
-
-def scrape_latest_chapter(url, site_path):
-    """Input: A url string and a 'site_path' which is essentially a Json object. (A list of dictionaries. The dictionaries are html tags and are arranged in a specific order.)
-    Output: A string scraped from the provided url that is the latest chapter of the manga at that url."""
-    chapter = BSoup(requests.get(url, timeout=30).text)
-    for tag in site_path:
-        if tag['TagClass'] != None:
-            chapter = chapter.find(tag['TagName'], {'class':tag['TagClass']})
+class CheckMangaUtil:
+    def __init__(self, db_name):
+        """Connects to DB, generates cache for site paths, and generates cursor object."""
+        self.con = sqlite3.connect(db_name)        
+        self.con.row_factory = sqlite3.Row
+        self.cur = self.con.cursor()
+        self.site_path_cache= {}
+    def get_entry_info(self, title):
+        """Input: A string representing the Manga title.
+        Output: Returns a dictionary of the manga's info from the db, keys to the dict are columns of the db."""
+        self.cur.execute("SELECT * FROM manga WHERE Title = (?)", [title])
+        entry = self.cur.fetchone()
+        return dict(entry)
+    def scrape_latest_chapter(self, url, site_path):
```

Comment:
PEP8 - single newline between funcs in a class

Reply:
Gotcha